src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -2288,4 +2289,36 @@ public void removeAllTenantMonitors(String tenantId, boolean sendEvents) {
       monitorRepository.deleteAllByTenantId(tenantId);
     }
   }
+
+  public RenderedMonitorTemplate renderMonitorTemplate(UUID monitorId, String resourceId,
+      String tenantId) {
+    //Get existing monitor
+    Monitor monitor = getMonitor(tenantId, monitorId).orElseThrow(
+        () -> new NotFoundException(String.format("No policy monitor found for %s", monitorId)));
```

Comment:
Sorry, I missed this typo before
```suggestion
        () -> new NotFoundException(String.format("No monitor found for %s", monitorId)));
```